lms/static/sass/views/_login-register.scss

```diff
@@ -111,51 +97,35 @@ $sm-btn-linkedin: #0077b5;
 
             .text {
                 position: relative;
-                background: white;
-                padding: 0 10px;
-                z-index: 6;
+                top: -2px; // Aligns center of text with center of line (CR)
+                padding: 0 $baseline;
+                background: $white;
                 text-transform: none;
-                font-size: 0.7em;
-                font-weight: 600;
+                z-index: 6;
             }
         }
     }
 
     .nav-btn {
-        margin: 0 28px;
-        padding: 7px 0;
-        width: calc( 100% - 56px );
-        border: 3px solid $m-blue-d5;
-        border-radius: 5px;
-        color: $m-blue-d5;
-        box-shadow: none;
-        text-shadow: none;
+        @extend %btn-secondary-blue-outline;
+        width: 100%;
```

Comment:
I think this is too wide for a secondary button. Can we make it 90% so it lines up with the sign in one?